users_ldap_groups/users_ldap_groups.py

```diff
@@ -0,0 +1,106 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    OpenERP, Open Source Management Solution
+#    This module copyright (C) 2012 Therp BV (<http://therp.nl>).
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+from openerp import models
+from openerp import fields
+from openerp import api
+import logging
+import users_ldap_groups_operators
+import inspect
+
+
+class CompanyLDAPGroupMapping(models.Model):
+    _name = 'res.company.ldap.group_mapping'
+    _rec_name = 'ldap_attribute'
+    _order = 'ldap_attribute'
+
+    def _get_operators(self):
```

Comment:
I've never been really happy with that, did you consider reimplementing the operators with models.AbstractModel?